src/qml/filters/gpstext/ui.qml

```diff
@@ -0,0 +1,815 @@
+					implicitWidth: 20
+					implicitHeight: 20
+					onClicked: { set_sec_offset_to_textfields(filter.get('videofile_timezone_seconds')) }
+				}
+				Shotcut.Button {
+					icon.source: 'qrc:///icons/dark/32x32/media-skip-backward'
+					Shotcut.HoverTip { text: 'Sync start of GPS to start of video file' +
+						'\nNote: use this if you started GPS and video recording at the same time' }
+					implicitWidth: 20
+					implicitHeight: 20
+					onClicked: { set_sec_offset_to_textfields(filter.get('auto_gps_offset_start')) }
+				}
+				Shotcut.Button {
+					icon.source: 'qrc:///icons/dark/32x32/media-playback-pause'
+					Shotcut.HoverTip { text: 'Sync start of GPS to current video file time at playhead' +
+						'\nNote: use this if you started GPS recording after video recording and filmed the moment of first fix' }
+					implicitWidth: 20
+					implicitHeight: 20
+					onClicked: { set_sec_offset_to_textfields(filter.get('auto_gps_offset_now')) }
+				}
+				Shotcut.UndoButton {
+					onClicked: {
+						set_sec_offset_to_textfields(0);
+						offset_secs.text = 0; 
+					}
+				}
+			}
+		}
+		
+		Label {
+			id: gps_sync_minor
+			text: qsTr('GPS minor offset')
+			Layout.alignment: Qt.AlignRight
+			leftPadding: 10
+			Shotcut.HoverTip { text: qsTr('This value is also added to gps time to sync with video time: ') }
+		} 
+		RowLayout {
+			Shotcut.SliderSpinner {
+				id: offset_slider
+				minimumValue: -60
+				maximumValue: 60
+				Layout.maximumWidth: 300
+				implicitWidth: 300
+				suffix: ' seconds'
+		        onValueChanged: {
+		            filter.set('minor_offset', value)
+		        }
+			}
+			Shotcut.Button {
+				icon.source: 'qrc:///icons/dark/32x32/lift'
+				Shotcut.HoverTip { text: qsTr('Move this offset into the major offset above') }
+				implicitWidth: 20 
+				implicitHeight: 20
+				onClicked: {
+					var new_offset = parseInt(Number(filter.get('minor_offset')), 10) + parseInt(Number(filter.get('major_offset')), 10); 
+					set_sec_offset_to_textfields(new_offset); 
+					offset_slider.value = 0;
+					filter.set('minor_offset', 0);
+				}
+			}
+			Shotcut.UndoButton {
+				Layout.alignment: Qt.AlignLeft
+				onClicked: { 
+					offset_slider.value = 0; 
+					filter.set('minor_offset', 0)
+				}
+			}
+		}
+				
+		Label {
+			topPadding: 10
+			bottomPadding: 5
+			text: qsTr('<b>Processing Options</b>')
+			Layout.columnSpan: 2
+		}
+		
+		Label {
+			text: qsTr('GPS smoothing')
+			leftPadding: 10
+			Layout.alignment: Qt.AlignRight
+			Shotcut.HoverTip { text: qsTr('Average nearby GPS points to smooth out errors.') }
+		}
+		RowLayout {
+			Shotcut.ComboBox {
+				implicitWidth: 300
+				id: combo_smoothing
+				model: 
+					[   qsTr('0 (raw data)'), 					//0
+				   		qsTr('1 (interpolate missing data)'),	//1
+		                qsTr('3 points'),						//2
+		                qsTr('5 points (recommended)'),			//3
```

Comment:
I do not think it is necessary to label something as recommended. But it is good to choose a default value that you think will work for most situations (as you have done).

Reply:
(I've moved them all except one of them which I'll do it in next commit after some more testing)
Question: when is _filter.isNew_ changing to false? I expected it to only execute once at the very begining of creating a new filter but it's not the case (after a window redraw or something it seems to enter it again).

Reply:
isNew only changes when reloading the filter UI. It is set once for each time the UI loads and indicates if the filter was added or already exists.

Reply:
Ok, it makes sense, I had a console.log() in a different area (independent of component.onCompleted) that gave me the wrong impression.